ci/scripts/integration_kartothek.sh

```diff
@@ -27,4 +27,5 @@ python -c "import pyarrow.parquet"
 python -c "import kartothek"
 
 pushd /kartothek
-pytest -n0 --ignore tests/cli/test_query.py
+# See ARROW-12314, test_load_dataframes_columns_raises_missing skipped because of changed error message
```

Comment:
Should we file an issue on their side to update the test?

Reply:
Yes, will do